project/ValidatePullRequest.scala

```diff
@@ -393,7 +393,7 @@ object AggregatePRValidation extends AutoPlugin {
         def hasTests(result: Tests.Output): Boolean = result.events.exists(e => hasExecutedTests(e._2))
 
         def printTestResults(result: KeyValue[Tests.Output]): Unit = {
-          write(s"Test result for '${showKey(result.key)}'")
+          write(s"Test result for `${showKey(result.key)}``")
```

Comment:
```suggestion
          write(s"Test result for `${showKey(result.key)}`")
```

Reply:
This still needs to be applied, right?

Reply:
Ah, good catch, wonder what happened there...